widgets/menu/menu.js

```diff
@@ -47,7 +47,7 @@ function menu({
   var containerNode = utils.getContainerNode(container);
   var usage = 'Usage: menu({container, facetName, [sortBy, limit, cssClasses.{root,list,item}, templates.{header,item,footer}, transformData, hideWhenResults]})';
 
-  if (container === null || facetName === null) {
+  if (!container || !facetName) {
```

Comment:
Nope, it's a line that had been forgotten in a previous refactoring I guess.

We used to pass default parameters as `container = null, facetName = null` and we then decided to simply pass `container, facetName` with no value and test if they were falsy.

Here we had done only half the job so I changed the second part :)